.prettierignore

```diff
@@ -0,0 +1 @@
+*.md
```

Comment:
I added this because I seem to be the only one who's environment is auto-formatting markdown. This stops prettier from doing that.

Reply:
I also have formatting on for these files. It is a bit annoying to have formatting changes as part of the PR, but I really like it for difficult things like formatting tables. See this README I recently had to edit and add items to the table, it was pure bliss to have the autoformatting.

https://github.com/woocommerce/woocommerce-admin/blob/main/packages/dependency-extraction-webpack-plugin/README.md

Reply:
@psealock I don't mind having auto formatting tbh. Lets just switch it on then and encourage everyone else to do so too.